vizro-core/src/vizro/models/_dashboard.py

```diff
@@ -109,26 +130,63 @@ def _make_page_layout(self, page: Page):
         # Shared across pages but slightly differ in content. These could possibly be done by a clientside
         # callback instead.
         page_title = html.H2(children=page.title, id="page_title")
-        navigation: _NavBuildType = cast(Navigation, self.navigation).build(active_page_id=page.id)
-        nav_bar = navigation["nav_bar_outer"]
-        nav_panel = navigation["nav_panel_outer"]
+        nav_content: _NavBuildType = cast(Navigation, self.navigation).build(active_page_id=page.id)
+        nav_bar = nav_content["nav_bar_outer"]
+        nav_panel = nav_content["nav_panel_outer"]
 
         # Different across pages
         page_content: _PageBuildType = page.build()
         control_panel = page_content["control_panel_outer"]
-        component_container = page_content["component_container_outer"]
-
-        # Arrangement
-        header = html.Div(children=[page_title, theme_switch], className="header", id="header_outer")
-        nav_control_elements = [dashboard_title, nav_panel, control_panel]
-        nav_control_panel = (
-            html.Div(nav_control_elements, className="nav_control_panel")
-            if any(not getattr(element, "hidden", False) for element in nav_control_elements)
-            else None
+        components = page_content["component_container_outer"]
+        return {
```

Comment:
Unless there's a good reason not to do so, I'd suggest doing this as an `html.Div` directly rather than a dictionary. Indexing would then be by the relevant `id` like `["dashboard-title"]`.